src/index.html

```diff
@@ -22,6 +22,456 @@
+            class="navigation__logo"
+          />
+        </div>
+      </div>
+      <nav class="navigation__nav">
+        <ul class="navigation__list">
+          <li class="navigation__item">
+            <a
+              href="#"
+              class="navigation__link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              href="#"
+              class="navigation__link"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              href="#"
+              class="navigation__link"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              href="#"
+              class="navigation__link"
+              data-qa="hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              href="#"
+              class="navigation__link"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              href="#"
+              class="navigation__link"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              href="#"
+              class="navigation__link"
+            >
+              PHOTO
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              href="#"
+              class="navigation__link"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <article
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__top">
+          <img
+            src="images/imac.jpeg"
+            class="card__top-img"
+            alt="imac"
+          />
+        </div>
+
+        <div class="card__bottom">
+          <h2 class="card__bottom-title">
+            APPLE A1419 iMac 27" Retina
+            <br />
+            5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__bottom-code">Product code: 195434</p>
+
+          <div class="card__bottom-stars">
+            <div class="stars stars--4">
+              <div class="stars__item"></div>
+              <div class="stars__item"></div>
+              <div class="stars__item"></div>
+              <div class="stars__item"></div>
+              <div class="stars__item"></div>
+            </div>
+
+            <p class="card__bottom-text">Reviews: 5</p>
+          </div>
+
+          <div class="card__bottom-cost">
+            <div class="card__bottom-price">Price:</div>
+            <div class="card__bottom-cost">$2,199</div>
+          </div>
+          <a
+            class="card__bottom-button"
```

Comment:
Ensure that the class name `card__bottom-button` matches the one in your SCSS file. Previously, there was a typo `card__bottom-botton` which needs to be corrected.